packages/math-input/src/components/aphrodite-css-transition-group/util.ts

```diff
@@ -0,0 +1,103 @@
+import {entries} from "@khanacademy/wonder-stuff-core";
+import {StyleSheet, css} from "aphrodite";
+
+import type {InAnimationStyles} from "./types";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+import type {CSSProperties} from "aphrodite";
+
+function flatten(list?: StyleType): Array<CSSProperties> {
+    const result: Array<CSSProperties> = [];
+
+    if (!list) {
+        return result;
+    }
+
+    if (Array.isArray(list)) {
+        for (const item of list) {
+            result.push(...flatten(item));
+        }
+    } else {
+        // @ts-expect-error - TS2345 - Argument of type 'CSSProperties | NestedArray<CSSProperties | Falsy>' is not assignable to parameter of type 'CSSProperties'.
+        result.push(list);
+    }
```

Comment:
I wonder why it thinks it's isn't a TypeScript error. 🤔

Reply:
It doesn't show up locally!  I wonder if this is a TypeScript mismatch.  I'll investigate.